modules/messages/client/components/ThreadMessage.js

```diff
@@ -70,30 +70,34 @@ export default function ThreadMessage({ message, user }) {
     return otherUser._id === user._id;
   }
 
+  const deletedUser = !message.userFrom.username;
+
   return (
     <MessageContainer message={message}>
       <div className="message-main">
         <div className="message-meta">
           {isMe(message.userFrom) ? (
             <span>{t('You')}</span>
-          ) : (
+          ) : !deletedUser ? (
             <a href={`/profile/${message.userFrom.username}`}>
               {message.userFrom.displayName}
             </a>
+          ) : (
+            <span>Unknown User</span>
```

Comment:
What do you think about doing "Unknown member" here instead, to be more in line with language across the site.

Reply:
Oh and needs translation function.

Reply:
Fixed in https://github.com/Trustroots/trustroots/pull/633/commits/756fd6feb7d636e4546f93bc1d0ea20009f32b35